pkg/payload/task.go

```diff
@@ -192,6 +192,9 @@ func SummaryForReason(reason, name string) string {
 	case "ImageVerificationFailed":
 		return "the image may not be safe to use"
 
+	case "UpgradeInProgress":
+		return "an upgrade is in progress"
```

Comment:
We use “update” over “upgrade”.

Might suggest “an upgrade is already in progress”.